WorkcraftCore/src/org/workcraft/plugins/shared/tasks/ExternalProcessTask.java

```diff
@@ -30,7 +30,7 @@
     public ExternalProcessTask(List<String> args, File workingDir) {
         this(args, workingDir, false, false);
     }
-
+    
```

Comment:
Hm. We have a regex that forbids trailing whitespaces, in theory. Don't merge this yet, I want to look into this.